tests/terraform/checks/resource/openstack/example_FirewallRuleSetDestinationIP/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "openstack_fw_rule_v1" "fail" {
+  name             = "my_rule_world"
+  description      = "let anyone in"
+  action           = "allow"
+  protocol         = "tcp"
+  destination_port = "22"
+  enabled          = "true"
+  # destination_ip_address = "10.0.0.1"
+}
+
+resource "openstack_fw_rule_v1" "pass" {
+  name             = "my_small_world"
+  description      = "let anyone in"
+  action           = "allow"
+  protocol         = "tcp"
+  destination_port = "22"
+  enabled          = "true"
+  destination_ip_address = "10.0.0.1"
```

Comment:
what happens, if I put in `0.0.0.0` is it then open to everyone? The documentation also states that you can a subnet os this `0.0.0.0/0` could be possible?

Reply:
good catch. fixed.